cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
I'd rather not because it can have adverse performance implications, and we're just doing this to get the tests to work, so I don't really care about a leaky abstraction.